diff --git a/src/main/java/com/wcs/poker/hand/HandRankingService.java b/src/main/java/com/wcs/poker/hand/HandRankingService.java
index a9e7918..1980af0 100644
--- a/src/main/java/com/wcs/poker/hand/HandRankingService.java
+++ b/src/main/java/com/wcs/poker/hand/HandRankingService.java
@@ -1,7 +1,6 @@
 package com.wcs.poker.hand;
 
 import com.wcs.poker.gamestate.Card;
-import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
